### ISSUE TITLE
PyPI Package name deprecation Warning

### ISSUE THREAD

**Issue by feanil** [openedx/code-annotations#127]
Got this e-mail from PyPI.

> This email is notifying you of an upcoming deprecation that we have determined may affect you as a result of your recent upload to 'code-annotations'.
> 
> In the future, PyPI will require all newly uploaded source distribution filenames to comply with [PEP 625](https://peps.python.org/pep-0625/). Any source distributions already uploaded will remain in place as-is and do not need to be updated.
> 
> Specifically, your recent upload of 'code-annotations-1.8.2.tar.gz' is incompatible with PEP 625 because it does not contain the normalized project name 'code_annotations'.
> 
> In most cases, this can be resolved by upgrading the version of your build tooling to a later version that supports PEP 625 and produces compliant filenames.
> 
> If you have questions, you can email [admin@pypi.org](mailto:admin@pypi.org) to communicate with the PyPI [admin@pypi.org](mailto:admin@pypi.org) to communicate with the PyPI administrators.

**Comment by feanil** [comment]
@bmtcril FYI

**Comment by bmtcril** [comment]
Got it, thanks. I'll take care of it before the next release, which should contain a number of fun and interesting changes anyway.

**Comment by bmtcril** [comment]
Or it may already be fixed since a huge amount of requirements got updated yesterday.

**Comment by bmtcril** [comment]
@feanil I think this was fixed in today's 2.0.0 release:

```
Uploading code_annotations-2.0.0-py2.py3-none-any.whl
Uploading code_annotations-2.0.0.tar.gz
```

Have you gotten any more warnings about it? If not, can you close this out?
